flight/Modules/UAVOMavlinkBridge/UAVOMavlinkBridge.c

```diff
@@ -125,7 +123,8 @@ static int32_t uavoMavlinkBridgeInitialize(void) {
 		module_enabled = true;
 		updateSettings();
 
-		serial_buf = PIOS_malloc(MAVLINK_MAX_PACKET_LEN);
+		mav_msg = PIOS_malloc(sizeof(*mav_msg));
```

Comment:
For core code like drivers we do an assert, but for optional modules it looks like we often just don't start the module, e.g. UAVOFrSkySensorHubBridge, UAVOHottBridge. I agree with throwing an assert(0) in core code like drivers, but for optional modules it seems reasonable to flag an error and disable the module, allowing the user to correct it without losing all the module and board settings.